src/VisualStudio/Core/Def/Implementation/SymbolSearch/SymbolSourceProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.VisualStudio.Language.Intellisense.SymbolSearch;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.SymbolSearch
+{
+    [Export(typeof(ISymbolSourceProvider))]
+    [ContentType(ContentTypeNames.RoslynContentType)]
+    [Name("Roslyn symbol source")]
```

Comment:
This is consumed in other places as well. Please add a const.

Reply:
renamed per Cyrus' comment